src/state/slices/postsSlice.ts

```diff
@@ -106,22 +106,27 @@ export const postsSlice = createSlice({
       if (action.payload === state.expandedPost.postId) {
         const isLikedByCurrentUser = !state.expandedPost.isLikedByCurrentUser;
         state.expandedPost.isLikedByCurrentUser = isLikedByCurrentUser;
-        isLikedByCurrentUser
-          ? state.expandedPost.numberOfLikes++
-          : state.expandedPost.numberOfLikes--;
+        if (isLikedByCurrentUser) {
```

Comment:
I've noticed a few turnary statements replaced by if else statements. Is this something the formatter has done or your own edits?

Reply:
This is a [rule](https://typescript-eslint.io/rules/no-unused-expressions/) in `typescript-eslint`. I think the point is that unused expressions like this can get confusing? Sort of a best practice thing I think. I can't find a solid answer but I think people like using ternaries for assignment operations and not side effects like increment/decrement or function calls

We can disable the rule if we want, or make it allow for ternaries